flutter-idea/src/io/flutter/run/daemon/DaemonConsoleView.java

```diff
@@ -75,6 +83,10 @@ public void print(@NotNull String text, @NotNull ConsoleViewContentType contentT
     }
   }
 
+  public void maybeSendErrorsToAnalytics() {
```

Comment:
I would remove this method. the name is confusing and the operation should be fast enough to just execute on each print so no need to add some possibly error prone caching.